src/Catel.Core/CoreModule.cs

```diff
@@ -49,7 +49,8 @@ public void Initialize(IServiceLocator serviceLocator)
             serviceLocator.RegisterType<IXmlSerializer, XmlSerializer>();
             serviceLocator.RegisterType<IXmlNamespaceManager, XmlNamespaceManager>();
             serviceLocator.RegisterType<ISerializationManager, SerializationManager>();
-            serviceLocator.RegisterType<IObjectAdapter, ObjectAdapter>();
+            serviceLocator.RegisterType<Catel.Runtime.Serialization.IObjectAdapter, Catel.Runtime.Serialization.ObjectAdapter>();
+            serviceLocator.RegisterType<Catel.Data.IObjectAdapter, Catel.Data.ReflectionObjectAdapter>();
```

Comment:
For consistency, these should probably not be referencing by the full namespace, unless this is a transitional PR.